app/views/payments/_payment.html.haml

```diff
@@ -0,0 +1,35 @@
+.div
+  .col-md-12.table-responsive
+    %table.table.table-hover
+      %thead
+        %tr
+          %th Ticket
+          %th Quantity
+          %th Price
+          %th Total
+      %tbody
+      - @unpaid_ticket_purchases.each do |ticket|
+        %tr
+          %td
+            = ticket.title
+          %td
+            = ticket.quantity
+          %td
+            = humanized_money_with_symbol ticket.price
+          %td
+            = humanized_money_with_symbol ticket.quantity * ticket.price
+
+= form_tag conference_payments_path do
+  %ul.price-tags.pull-right
+    %li
+      %a
+        = humanized_money_with_symbol @total_amount_to_pay
+  %script.stripe-button{ src: "https://checkout.stripe.com/checkout.js",
+        data: { amount: @total_amount_to_pay.cents,
+          currency: @total_amount_to_pay.currency,
+          image: image_url('OSEM_ICON.jpg'),
+          name: ENV['OSEM_NAME'] || 'OSEM',
+          description: "book your tickets",
+          key: Rails.application.secrets.stripe_publishable_key,
```

Comment:
But then why does the `stripe.rb` initializer try to read environment variables and set them in `Rails.configuration`?